lapis-e2e/testData/multiSegmented/input_file.ndjson

```diff
@@ -1,4 +1,4 @@
-{"metadata":{"primaryKey": "key_0", "date": "2021-03-18", "country": "Switzerland"},"nucleotideInsertions": {"L": [], "M":  [], "S":  []}, "aminoAcidInsertions":  {"RdRp":  [], "NP":  [], "GPC":  []}, "alignedNucleotideSequences":  {"L": "ACTCT", "M": "CGGG", "S": "AAA"}, "unalignedNucleotideSequences":  {"L": null, "M": null, "S":  null}, "alignedAminoAcidSequences": {"RdRp": "MDFL*", "GPC": "MS*","NP": "MEN*"}}
+{"metadata":{"primaryKey": "key_0", "date": "2021-03-18", "country": "Switzerland"},"nucleotideInsertions": {"L": [], "M":  [], "S":  []}, "aminoAcidInsertions":  {"RdRp":  [], "NP":  [], "GPC":  []}, "alignedNucleotideSequences":  {"L": "ACTCT", "M": "CGGG", "S": "AAA"}, "unalignedNucleotideSequences":  {"L": "L_key_0", "M": null, "S":  null}, "alignedAminoAcidSequences": {"RdRp": "MDFL*", "GPC": "MS*","NP": "MEN*"}}
```

Comment:
could we maybe choose a valid example here? I was confused when reading the tests to see `L_key_0` as the nucleotide sequence. Can't we set aligned as "ACTNCT" and unaligned as "ACTCT"?

Reply:
Hm, actually I like it when test data is a bit more speaking than "random characters". Test data doesn't necessarily need to make sense, it needs to work and it should reveal the intention of the test. But I can change it if you find it too confusing.